src/fetch_tools/commands/debug_snapshot.py

```diff
@@ -35,8 +35,8 @@
 # All commands requiring sudo must be added in main
 commands = {"dpkg_fetch":"COLUMNS=200 dpkg -l ros-indigo-fetch-*",
             "dpkg_all":"COLUMNS=200 dpkg -l",
```

Comment:
Yeah when you don't have an actual terminal the column width gets messed up on dpkg. You need that to ensure you can actually read it in the text file that we generate.